RequestInfoSvcs/RequestInfoManager/RequestLogLoader.cpp

```diff
@@ -274,6 +283,17 @@ namespace RequestInfoSvcs
           log_errors_->report_error(
             Generics::ActiveObjectCallback::ERROR, ostr.str());
         }
+           std::regex r(".([0-9A-Za-z]+).[0-9a-z]+$");
```

Comment:
Use name_info.base_name here intead ext. Remove regexp (and its regular recompilation)